examples/informers/Program.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace informers
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
```

Comment:
can this example be a non asp.net app
I mean pure console app

Reply:
This is not an aspnet - it's `Microsoft.NET.Sdk.Worker`. The included example is of a simple controller, and as such is expected to work as a worker code. The informer code is also intended to be used primarily with DI.